openmc/cell.py

```diff
@@ -79,13 +79,12 @@ class Cell(object):
     translation : Iterable of float
         If the cell is filled with a universe, this array specifies a vector
         that is used to translate (shift) the universe.
-    offsets : ndarray
-        Array of offsets used for distributed cell searches
-    distribcell_index : int
-        Index of this cell in distribcell arrays
-    distribcell_paths : list of str
-        The paths traversed through the CSG tree to reach each distribcell
-        instance
+    paths : list of str
+        The paths traversed through the CSG tree to reach each cell
+        instance. This property is initialized by calling the
+        :meth:`Geometry.determine_paths` method.
```

Comment:
I personally prefer the latter option. A moment ago as I was trying to test this code out, I was importing the BEAVRS builder into Python and was wondering why it was taking so long (albeit 12-ish seconds) since I thought I had improved matters by removing the OpenCG intermediary. I think the runtime is largely dominated by the path determination...which is called regardless of whether one wants them or not (or is doing XML generation vs. data processing). A few seconds isn't a huge deal of course, though I would vote to make the user call `determine_paths()` if and only if they care about such things.